lib/configSchema.test.js

```diff
@@ -71,6 +71,7 @@ describe('#configSchema', () => {
             events: [],
             handler: 'someHandler',
             name: 'some-service-dev-someFunc',
+            someRequiredFunctionNumberProp: 123,
```

Comment:
Not sure if there is a better solution. This test started failing after I changeed `test/fixtures/configSchemaExtensions/test-plugin.js`

Reply:
It's ok, fixture got changed so test might need to be updated